src/main/resources/languages/fr-FR.json

```diff
@@ -246,7 +246,7 @@
       "send": "%s %s (niveau %s) ont été ajouté au message.\nContinuez d'ajouter plus d'objets ou utilisez '/sendmail finish' pour envoyer le message.",
       "invalid_arguments_please_use": "Arguments invalides.\n Veuillez utiliser '/sendmail %s'",
       "title": "<titre>",
-      "message": "🇺🇸<message>",
+      "message": "<message>",
```

Comment:
Just so you know, the linter will bring these marks back on identical strings next time it is run.